src/api/route-services/__mocks__/experiment.js

```diff
@@ -52,21 +52,30 @@ const mockGetProcessingConfig = jest.fn(() => new Promise((resolve) => {
 const mockUpdateProcessingConfig = jest.fn(
   () => new Promise((resolve) => {
     resolve({
-      cellSizeDistribution: {
-        M: {
+      processingConfig: {
+        cellSizeDistribution: {
+          filterSettings: { minCellSize: 10800, binStep: 200 },
+          enabled: true,
```

Comment:
@aerlaut is this the same format that you modified to recently in one of your changes of the UI for data integration config?

Reply:
`dimensionalityReduction` is changed to `dataIntegration` which has the following default state : 
```
dataIntegration: {
      dataIntegration: {
        method: 'seuratv4',
        methodSettings: {
          seuratv4: {
            numGenes: 2000,
            normalisation: 'logNormalise',
          },
        },
      },
      dimensionalityReduction: {
        method: 'rpca',
        numPCs: 30,
        variationExplained: 91,
        excludeGeneCategories: [],
      },
    }
```

the key `dimensionalityReduction` is removed. This change is actually outside of this task, it should be done when persisting `dataIntegration` config to database,